app/models/DeclarationType.scala

```diff
@@ -16,61 +16,32 @@
 
 package models
 
-import models.ProcedureType.{Normal, Simplified}
+import models.ProcedureType.Normal
 import models.reference.CountryCode
-import pages.{OfficeOfDeparturePage, ProcedureTypePage}
-import play.api.data.Form
-import uk.gov.hmrc.viewmodels._
 
-sealed abstract class DeclarationType(val code: String, asString: String) extends WithName(asString)
+sealed trait DeclarationType
 
-object DeclarationType extends Enumerable.Implicits {
+object DeclarationType {
 
-  case object Option1 extends DeclarationType("T1", "option1")
-  case object Option2 extends DeclarationType("T2", "option2")
-  case object Option3 extends DeclarationType("T2F", "option3")
-  case object Option4 extends DeclarationType("TIR", "option4")
+  case object Option1 extends WithName("T1") with DeclarationType
+  case object Option2 extends WithName("T2") with DeclarationType
+  case object Option3 extends WithName("T2F") with DeclarationType
+  case object Option4 extends WithName("T") with DeclarationType
+  case object Option5 extends WithName("TIR") with DeclarationType
 
   val t2Options = Seq(Option2, Option3)
 
   val values: Seq[DeclarationType] = Seq(
     Option1,
     Option2,
     Option3,
-    Option4
+    Option4,
+    Option5
   )
 
-  def radios(form: Form[_], userAnswers: UserAnswers)(): Seq[Radios.Item] = {
-
-    val field = form("value")
-    val itemsGB = Seq(
-      Radios.Radio(msg"declarationType.option1", Option1.toString),
-      Radios.Radio(msg"declarationType.option2", Option2.toString)
-    )
-    val itemsNISimplified = Seq(
-      Radios.Radio(msg"declarationType.option1", Option1.toString),
-      Radios.Radio(msg"declarationType.option2", Option2.toString),
-      Radios.Radio(msg"declarationType.option3", Option3.toString)
-    )
-    val itemsNINormal = Seq(
-      Radios.Radio(msg"declarationType.option1", Option1.toString),
-      Radios.Radio(msg"declarationType.option2", Option2.toString),
-      Radios.Radio(msg"declarationType.option3", Option3.toString),
-      Radios.Radio(msg"declarationType.option4", Option4.toString)
-    )
-    val countryCode = userAnswers.get(OfficeOfDeparturePage).map(_.countryId)
-    val items = (countryCode, userAnswers.get(ProcedureTypePage)) match {
-      case (Some(CountryCode("XI")), Some(Simplified)) => itemsNISimplified
-      case (Some(CountryCode("XI")), Some(Normal))     => itemsNINormal
-      case _                                           => itemsGB
+  def chooseValues(countryCode: Option[CountryCode], procedureType: Option[ProcedureType]): Seq[DeclarationType] =
```

Comment:
I assume this is correct and we only have two potential lists, one with 4 options, and one with 5